options/locale/locale_en-US.ini

```diff
@@ -118,6 +118,20 @@ footer = Footer
 footer.software = About Software
 footer.links = Links
 
+[editor]
+header.tooltip = Add heading
```

Comment:
`header` doesn't mean `Add heading` in English?

Reply:
I guess the key could be something like:

`toolbar.add_heading=.....`

`toolbar.list_ordered=.....`

or 

`toolbar.add_heading.tooltip=.....`

Reply:
What about something like `[editor].buttons.heading.tooltip`?

Reply:
Although I prefer "toolbar.xxx", I have no objection for either.

My thought is: in the "editor" component, there could be many other buttons, for example, "link" button (add a link, or link to another issue/PR?), "preview" tab/button, "save" button (save draft?),  etc. Normal buttons are usually different from toolbar buttons and these buttons may have different titles / tooltips.

But it's not that important, either is fine.